### PR TITLE
add(AMGEO): first lecture

### PR DESCRIPTION
**Pull request by lugggas**
Some stuff has been added from the book.

### CODE REVIEW

**Comment by LostInDarkMath** [comment]
Is there a PDF-File included? It would be nice to have the newest PDF files in the repository. That makes ist easierer for others to get our content.

**Comment by phijor** [comment]
But, but, but: git is not meant to store binary files that update frequently :o

We could easily build .pdfs once a week and use the "GitHub releases" feature to distribute them. Then one would not even need to clone the repo in order to get access to the transcripts.

Philipp Joram

File: AMGEO/chapter0.tex [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    Conversely, if $\liebracket$ is bilinear and $[x,y] = -[y,x]$, we get
+    \begin{align*}
+        \implies 2 [x,x] = 0\\
+        \implies [x,x] = 0,
+    \end{align*}
+    unless $\textrm{char } \F = 2$.
+\end{remark}
+
+\begin{notation}
+    If we just write $\F$ we mean an algebraically closed field with $\Q \subseteq \F$.
```
Comment (lukasjuhrich) [inline AMGEO/chapter0.tex]: I wasn't in the lecture, but out of curiosity, isn't that equivalent to saying that the characteristic must be zero? IIRC ℚ is initial in \underline{Fld} so the image is either infinite or the field has nonzero characteristic.

File: AMGEO/chapter0.tex [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+        \end{cases}. $$
+        Note also that this does not depend on the chosen basis, 
+        as one may show that for all $g \in \GL(V)$:
+        $$ \tr(gag^{-1}) = \tr(a). $$
+
+        If now $a,b \in \sl$ we get
+        \begin{align*}
+            \tr(ab) = \tr(ba) \implies \tr [a,b] = \tr (ab - ba) = \tr(ab) - \tr(ba) = 0. 
+        \end{align*}
+        So $\sl(V)$ is linear sub algebra.
```
Comment (lukasjuhrich) [inline AMGEO/chapter0.tex]: `sub algebra` → `subalgebra`

File: AMGEO/chapter0.tex [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    Consider
+    $$e_{ij} e_{kl} = \delta_{jk}e_{il}, $$ 
+    hence
+    $$[e_{ij}, e_{kl}] = \delta_{jk}e_{il} - \delta_{li} e_{kj}.$$
+    Note the coeffitients in front of each $e_{rs}$ are 1,0 or  $-1$.
+\end{remark}
+
+\begin{example}
+    The "'classical algebras"' are for $l \in \N$:
+    $$A_l, B_l, C_l, D_l. $$
```
Comment (lukasjuhrich) [inline AMGEO/chapter0.tex]: Did he actually call them `A,B,C,D`? Is that done anywhere else in literature? :D
Or is that motivated by literature?

**Comment by phijor** [comment]
I guess these are the canonical names. See https://en.wikipedia.org/wiki/Semisimple_Lie_algebra#Classification.

File: AMGEO/chapter0.tex (hunk 1 above)
Comment (lugggas) [reply AMGEO/chapter0.tex]: That's right.I add this as a note. Thanks.

File: AMGEO/chapter0.tex (hunk 3 above)
Comment (lugggas) [reply AMGEO/chapter0.tex]: I don't know, but I think it's related to the literature.

File: AMGEO/chapter0.tex (hunk 2 above)
Comment (lugggas) [reply AMGEO/chapter0.tex]: Sure.

**Comment by lugggas** [comment]
@phijor
Thank you for backup.

File: AMGEO/chapter0.tex (hunk 1 above)
Comment (lugggas) [reply AMGEO/chapter0.tex]: Done

File: AMGEO/chapter0.tex (hunk 2 above)
Comment (lugggas) [reply AMGEO/chapter0.tex]: Done.

**Pull request merged**